src/Network.js

```diff
@@ -13,6 +13,16 @@ import {accessor, assign, configPrep, constant, elem} from "d3plus-common";
 import * as shapes from "d3plus-shape";
 import {dataLoad as load, Viz} from "d3plus-viz";
 
+/**
+  @function constructAriaLabel
+  @desc Returns value for aria-label property of Shapes.
+  @private
+*/
+function constructAriaLabel(d, i) {
+  const validSize =  this._size === undefined ? "" : typeof this._size === "function" ? this._size(d) :  this._size;
+  return this._drawLabel(d, i) + validSize + ".";
```

Comment:
1. Because of how the `size( )` method is written, `this._size` will always be a function if set.
2. You always should be passing the index `i` to all functions as the second argument.
3. You are missing the comma and space that come between the drawLabel and the size. 

With all of that, this logic should look more like this:

```js
const validSize = this._size ? `, ${this._size(d, i)}` : "";
return `${this._drawLabel(d, i)}${validSize}.`;
```